WtWSMS/common/culture/name_lists/BP_east_germanic.txt

```diff
@@ -334,62 +334,46 @@ name_list_crimean_gothic = {
 	cadet_dynasty_names = {
-		"dynn_Baldemaring"
-		"dynn_Dagiling"
-		"dynn_Dietriching"
-		"dynn_Engelbrechting"
-		"dynn_Godigiseling"
+		"dynn_Victohali"
 		"dynn_Hasdingi"
-		"dynn_Rudging"
 		"dynn_Silingi"
-		"dynn_Simoning"
+		"dynn_Carini"
+		"dynn_Lugii"
+		"dynn_Lacringi"
+		"dynn_Harii"
+		"dynn_Helveconae"
+		"dynn_Manimi"
+		"dynn_Helisii"
+		"dynn_Nahanarvali"
 	}
 	
 	dynasty_names = {
-		"dynn_Baldemaring"
-		"dynn_Dagiling"
-		"dynn_Dietriching"
-		"dynn_Engelbrechting"
-		"dynn_Godigiseling"
+		"dynn_Victohali"
 		"dynn_Hasdingi"
-		"dynn_Rudging"
 		"dynn_Silingi"
-		"dynn_Simoning"
+		"dynn_Carini"
+		"dynn_Lugii"
+		"dynn_Lacringi"
+		"dynn_Harii"
+		"dynn_Helveconae"
+		"dynn_Manimi"
+		"dynn_Helisii"
+		"dynn_Nahanarvali"
 	}
 	
 	male_names = {
-		Adalbert Adam Adolf Albrecht Almerich Amadeus Andreas Anselm Arnold Arnulf Augustin 
-		Baldemar Baldewin Berengar Bernhard Berthold Bruno Burchard Caspar Christopher 
-		Diether Dietmar Dietpold Dietrich Dietwin Eberhard Eckhard Ehrenfried Eginolf Ekbert Emelrich Emich
-		Engelbrecht Erich Ernst Ezzo Folkhard Folkmar Franz Friedrich Gebhard Gelimer Georg
-		Gerhard Gerlach Giselbert Gottfried Gotthard Gottschalk Gotzelo Guntram GU_nzel Hartmann Hartwig 
-		Heinrich Herbert Hermann Hesso Hilderic Hugo Humbert Huneric Jakob Johann Karl Karloman
-		Konrad Kuno Lambert Leopold Lothar Ludolf Ludwig Luitpold Lutbert Magnus
-		Manfred Markward Martin Mathias Meinhard Michael Nikolaus Norbert Ordulf Otto
-		Peter Philipp Poppo Rudger Reinhard Rudolf Ruprecht Siegfried Sieghard Siegmund 
-		Simon Stefan Stilicho Theodorich Thietmar Thomas Udo Ulrich Viktor Waldemar Walram
-		Walther Welf Wenzel Werner Widukind Wigerich Wilhelm Wolfgang Wolfram
+		Amata Anduit Ansila Armogas Berimuth Blumarith Brandinus Cyrila
+		Gaisarith Gaisarix Gamuth Geilamir Geilarith Gibamunds Gintun Godagisl Gudulo Guntha Gunthamunds Guntharix Dagila Eigerith Etemunds 
+		Feva Fridubalth Fridila Fronimuth Ingila Iulioteus 
+		Hariarith Harifridus Hildica Hildirix Hohageis Hohamir Hunirix 
+		Maiorix Manno Merobaudes Muritta Obadus Pinta Rag Raginari Rapt
+		Scarila Siddinus Sigisteus Sifila Sindivult Silibudi Stilicho
+		Tanca Theudarix Thrasamunds Victorix Vifridus Vilimuth Visimar Vitarith Untancus Usclu
 	}
 	
 	female_names = {
-		A_lfrhildr AlfrI_D_r A_lvO_r A__sa A_slaug A__sta A_strI_D_r AuT_r
-		BjO_rg Bodil BO_thildr BergT_O_ra
-		CecilI_a
-		DrI_fa Dalla
-		ED_la
-		Freyja Freydi_s
-		GerT_r GlO_D_ GrI_ma Gunnhildr Gurli GuD_rU_n GrY_la GyrI_D_r GyT_a
-		HafrI_D_r Halla HO_lmfrI_D_r Huld
-		Iliana Inga IngfrI_D_r IngibjO_rg IngjerT_r IngirI_D_r
-		JO_runn
-		KrA_ka
-		ME_r MA_lmfrI_D_r
-		RI_kiza RO_gnfrI_D_r Ragnhildr
-		SA_ga SI_g Sif SigrI_D_r Skuld Svanhildr SignY_ SkjaldvO_r
-		T_O_ra T_O_rdI_s T_yri T_O_runn T_uriD_r
-		U_lfhildr Unnr
-		VigdI__s
-		Ylva Yrsa Y_r
+		Agisild Aviarica Berimutha Gilesa Dagilia Damira
+		Mannica Munifrida Siddina Triova Valilu Viliaruna Vitifrida
```

Comment:
See other comment on the insufficient number of female names (otherwise great job with this, the list is much better)

Reply:
> See other comment on the insufficient number of female names (otherwise great job with this, the list is much better)

I'm well aware about that issue and previously expressed my worries to LT-Rascek, but he thinks there is just enough. Although, if you wish, I may add some Gothic names.

Reply:
Yes, I think that would be good if you have the time to do it.